server/modules/analysis/api/processingChainApi.js

```diff
@@ -167,6 +167,24 @@ export const init = app => {
     },
   )
 
+  // ====== READ - Chain Data
+
```

Comment:
Create rChainApi and use prefix /survey/:surveyId/rChain/step

Reply:
ok